### ISSUE TITLE
Check and fix study date range

### ISSUE THREAD

**Issue by vvcb** [LTHTR-DST/hdruk_avoidable_admissions#6]
This is linked to <https://github.com/MattStammers/hdruk_avoidable_admissions_collaboration_docs/issues/3> and email conversations around the correct study date range. Once agreed and if different from what is in place at the moment, the following code will need to be modified.

- <https://github.com/LTHTR-DST/hdruk_avoidable_admissions/blob/dev/avoidable_admissions/data/validate.py#L56>
- <https://github.com/LTHTR-DST/hdruk_avoidable_admissions/blob/dev/avoidable_admissions/data/validate.py#L173>

**Comment by MattStammers** [comment]
I think this is now correct

**Comment by MattStammers** [comment]
no this is not correct but I can't merge it  with the ticket above (they need to be joined and this one closed

**Comment by vvcb** [comment]
Duplicate of #16
